core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -844,6 +945,39 @@ extern "C" fn new_show_info(n_args: usize, args: *const Obj, kwargs: *mut Map) -
     unsafe { util::try_with_args_and_kwargs(n_args, args, kwargs, block) }
 }
 
+extern "C" fn new_show_mismatch(n_args: usize, args: *const Obj, kwargs: *mut Map) -> Obj {
+    let block = move |_args: &[Obj], _kwargs: &Map| {
+        let title: StrBuffer = "Address mismatch?".into();
+        let description: StrBuffer = "Please contact Trezor support at".into();
+        let url: StrBuffer = "trezor.io/support".into();
+        let button = "QUIT";
+
+        let icon = BlendedImage::new(
```

Comment:
Good idea, I added a constructor for this in b11c13260cb04c2443ee81fbd359602f70fd23e1. It would be easy to replace if we have a reason to write code for painting single non-blended image.